python/python/glide/async_commands/cluster_commands.py

```diff
@@ -291,31 +296,31 @@ async def dbsize(self, route: Optional[Route] = None) -> int:
         return cast(int, await self._execute_command(RequestType.DBSize, [], route))
 
     async def echo(
-        self, message: str, route: Optional[Route] = None
-    ) -> TClusterResponse[str]:
+        self, message: TEncodable, route: Optional[Route] = None
+    ) -> TClusterResponse[bytes]:
         """
         Echoes the provided `message` back.
 
         See https://redis.io/commands/echo for more details.
 
         Args:
-            message (str): The message to be echoed back.
+            message (TEncodable): The message to be echoed back.
             route (Optional[Route]): The command will be routed to a random node, unless `route` is provided,
             in which case the client will route the command to the nodes defined by `route`.
 
         Returns:
-            TClusterResponse[str]: The provided `message`.
+            TClusterResponse[bytes]: The provided `message`.
             When specifying a route other than a single node, response will be:
-            {Address (str) : response (str) , ... } with type of Dict[str, str].
+            {Address (bytes) : response (bytes) , ... } with type of Dict[bytes, bytes].
 
         Examples:
-            >>> await client.echo("Glide-for-Redis")
-                'Glide-for-Redis'
-            >>> await client.echo("Glide-for-Redis", AllNodes())
-                {'addr': 'Glide-for-Redis', 'addr2': 'Glide-for-Redis', 'addr3': 'Glide-for-Redis'}
+            >>> await client.echo(b"Glide-for-Redis")
+                b'Glide-for-Redis'
+            >>> await client.echo(b"Glide-for-Redis", AllNodes())
```

Comment:
```suggestion
            >>> await client.echo("Glide-for-Redis", AllNodes())
```

I want to have an example of both str and bytes